db/migrate/20251021145145_create_table_user_subscription_targets.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+Sequel.migration do
+  up do
+    if TradeTariffBackend.uk?
+      unless Sequel::Model.db.table_exists?(Sequel[:user_subscription_targets].qualify(:public))
+        create_table Sequel[:user_subscription_targets].qualify(:public), id: :uuid do
```

Comment:
We had an issue with this previously as I don't think this actually created the id as a UUID. However, I'm not sure there is a need for the primary key to be a UUID anyway, so probably OK just to remove?
```suggestion
        create_table Sequel[:user_subscription_targets].qualify(:public) do
```